### DISCUSSION TITLE
OpenAPI / Swagger JSON for speech2text avaiblable?

### DISCUSSION THREAD

**Discussion by chris299** [deepgram/community#669] [General help] [answered]
Hi,
is there an OpenAPI / Swagger JSON Definition for the speech2text API available?
I did not find it in the docs at https://developers.deepgram.com/reference/listen-file
kind regards
Christoph

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by lukeocodes** [comment, marked as answer]
Something I am working on this quarter, but not publicly available yet. If you'd like to join our Discord, I'll keep you up to date on it as I am working on it.
